pkg/recommender.py

```diff
@@ -441,20 +443,57 @@ def _schedule_evt(self):
     Send the morning message at 10 am
     '''
     next_morning = ''
+    #Default message time
+    morn_hour = 10
+    morn_min = 0
+    ev_hour = 23
+    ev_min = 0
+
+    #get start time from deployment
+    try:
+      con = sqlite3.connect('C:/Users/Obesity_Project/Desktop/Patient-Caregiver Relationship/Patient-Caregiver-Relationship/DeploymentInformation.db')
+      cursorObj =  con.cursor()
+
+      table_name = 'RESIDENTS_DATA'
+      #select the latest deploymnet by ordering table by created date
+      cursorObj.execute("SELECT * FROM " + table_name + " ORDER BY created_date ASC LIMIT 1")
+      rows = cursorObj.fetchall()[0][11] # extract only start time
+      start_hour = int(rows[0:2])
+      start_minute = int(rows[3:5])
```

Comment:
would the time possibly be `9:00` instead of `09:00`? 

if the `rows` is a string, you can do `split`
```
start_hour, start_min = [int(t) for t in rows.split(':')]
```